acra-censor/handlers/denyall_handler.go

```diff
@@ -20,7 +21,7 @@ func NewDenyallHandler() *DenyAllHandler {
 // CheckQuery blocks any input query
 func (handler *DenyAllHandler) CheckQuery(sqlQuery string, parsedQuery sqlparser.Statement) (bool, error) {
 	// deny any query and stop further checks
-	handler.logger.Errorf("Query has been block by Denyall handler")
+	handler.logger.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCensorQueryIsNotAllowed).Errorf("Query has been block by Denyall handler")
```

Comment:
Sounds a bit weird, but we started using "deny" everywhere, so..

```suggestion
	handler.logger.WithField(logging.FieldKeyEventCode, logging.EventCodeErrorCensorQueryIsNotAllowed).Errorf("Query has been denied by Denyall handler")
```